compiler/modules/front/expr.bal

```diff
@@ -574,6 +581,38 @@ function codeGenArithmeticBinaryExpr(ExprContext cx, bir:BasicBlock bb, bir:Arit
     return { result, block: bb };
 }
 
+function intArithmeticOpNeverPanics(bir:ArithmeticBinaryOp op, bir:Operand[] operands) returns bir:INSN_INT_NO_PANIC_ARITHMETIC_BINARY|bir:INSN_INT_ARITHMETIC_BINARY {
```

Comment:
A function with this name needs to return boolean.

Reply:
fixed